cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -1073,6 +1080,7 @@ impl ManagedEnvironment {
 
     pub fn pull(&mut self, force: bool) -> Result<(), ManagedEnvironmentError> {
```

Comment:
Currently I return an error variant if the environment is already up-to-date and match on that.
Alternatively I can change this signature to `Result<PullState, ManagedEnvironmentError>` where PullState is

```
enum PullState {
	Updated
	UpToDate
}
```

(or a bool, but that doesn't read as clearly IMO).

Reply:
The error turned out to interact weirdly with `push`.